.idea/compiler.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please remove this folder (.idea) from repo

Reply:
deleted